app/assets/javascripts/solidus_paypal_braintree/paypal_button.js

```diff
@@ -41,6 +41,7 @@ SolidusPaypalBraintree.PaypalButton.prototype.initializeCallback = function() {
   this._paymentMethodId = this._client.paymentMethodId;
 
   this._client.getPaypalInstance().loadPayPalSDK({
+    "client-id": this._environment === "sandbox" ? "sb" : null,
```

Comment:
What about configuring this at `_paypalOptions` level instead? I see that a lot of other configurations are done this way.

Reply:
`this._environment` does come from the `paypalOptions`, it just was never being used here.